src/deepak/ArrayIdenticalOrNot.java

```diff
@@ -0,0 +1,37 @@
+package deepak;
+
+public class ArrayIdenticalOrNot {
+	static boolean flag;
+	void checkIdentical(int[] ar1, int[] ar2) {
+		
+		boolean flag = false;
+	
+		if(ar1.length == ar2.length) {
+			
+			for(int index=0; index < ar1.length; index++) {
+				
+				if(ar1[index] == ar2[index])
+					flag = true;
+				else
+					flag = false;
+			}
+		} 
+		
+		if(flag == true) {
```

Comment:
We should not check == with true, look not logical
because that flag value already have some value like true/false.